### ISSUE TITLE
Add flag to add a tag to hyperdrive

### ISSUE THREAD

**Issue by RangerMauve** [RangerMauve/hyperdrive-publisher#4]
A `--tag` CLI flag with a name, the `createTag` command will run after the hyperdrive sync so that people could use the URL.

Should include an example in the README

**Comment by benhylau** [comment]
@pfrazee does Beaker have support for fetching versioned URLs?

e.g. `hyper://blog.mauve.moe+300`

where `300` is not the latest version of the drive, and will come in as the `etag` of the response header.

**Comment by pfrazee** [comment]
@benhylau yeah the `+{number}` syntax does work. We need to add the `+{tag}` behavior now

**Comment by RangerMauve** [comment]
Regarding tags in URLs,

Here's the code we use in dat-fetch/Agregore BTW. https://github.com/RangerMauve/dat-fetch/blob/bff217a9f3a2347af82ddd43f24442401221f878/index.js#L115

There's a tricky case where tags that are plain numbers mess up the detection. I don't think Gateway supports it yet either.

**Comment by pfrazee** [comment]
Yeah simple enough! I’d suggest we just not support integer tags.
